scripts/winsetup/Yarn.wxs

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+	<Product Id="*" Name="Yarn-Next" Language="1033" Version="$(var.YarnVersion)" Manufacturer="Yarn Contributors" UpgradeCode="7cd8acb0-ce9f-4679-8e81-4eb9a69bf772">
+		<Package InstallerVersion="200" Compressed="yes" InstallScope="perMachine" />
+
+		<MajorUpgrade DowngradeErrorMessage="A newer version of [ProductName] is already installed." />
+		<MediaTemplate EmbedCab="yes" />
+
+		<!-- Ensure Node.js is installed. Can be either 32-bit or 64-bit, we don't particularly care -->
+		<!--<Property Id="NODEVERSION">
+			<RegistrySearch Id="NodeVersionReg32bit" Root="HKLM" Key="SOFTWARE\Node.js" Name="Version" Type="raw" Win64="no" />
+			<RegistrySearch Id="NodeVersionReg64bit" Root="HKLM" Key="SOFTWARE\Node.js" Name="Version" Type="raw" Win64="yes" />
+		</Property>
+		<Condition Message="Yarn requires Node.js 4.0 or higher to be installed">NODEVERSION</Condition>-->
+
+		<!-- If Yarn is already installed, default to the current installation directory -->
+		<Property Id="INSTALLDIR">
+			<RegistrySearch Id="YarnExistingInstallPath" Root="HKLM" Type="raw" Key="SOFTWARE\Yarn-Next" Name="InstallDir" />
+		</Property>
+
+		<Feature Id="MainFeature" Title="Yarn-Next" Level="1">
+			<ComponentGroupRef Id="YarnFiles" />
+			<ComponentRef Id="YarnRegistryAndPath" />
+		</Feature>
+
+		<Directory Id="TARGETDIR" Name="SourceDir">
+			<Directory Id="ProgramFilesFolder">
+				<Directory Id="INSTALLDIR" Name="Yarn-Next" />
+			</Directory>
+		</Directory>
+
+		<DirectoryRef Id="INSTALLDIR">
+			<Component Id="YarnRegistryAndPath">
+				<RegistryValue Root="HKLM" Key="SOFTWARE\Yarn-Next" Type="string" Name="InstallDir" Value="[INSTALLDIR]" KeyPath="yes" />
+				<Environment
+					Id="Path"
+					Name="PATH"
+					Value="[INSTALLDIR]bin\"
+					Permanent="no"
+					Part="last"
+					Action="set"
+					System="yes"
+				/>
+				<Environment
+					Id="BinPath"
+					Name="PATH"
+					Value="[LocalAppDataFolder]Yarn\bin"
```

Comment:
@arcanis - When global packages are installed with Berry, where do they end up? This is the path for Yarn v1 (`%LocalAppData%\Yarn\bin`) but I wasn't sure what the path is for v2.

Reply:
At the moment the global folder is `LocalAppData\Yarn\Berry`. There isn't a "global binary directory" at the moment as I'd like to see how far we can go with [`yarn dlx`](https://next.yarnpkg.com/cli/dlx), but a `bin` subdirectory would be a safe guess.